### ISSUE TITLE
correct.json should be a part of `ValidBlocks`

### ISSUE THREAD

**Issue by gurukamath** [ethereum/legacytests#2]
The `correct.json` test which is currently a part of the `InvalidBlocks` folder should instead be moved to the `ValidBlocks` since this contains valid blocks.

`legacytests/Constantinople/BlockchainTests/InvalidBlocks/bcUncleHeaderValidity/correct.json`

**Comment by winsvega** [comment]
yes. the idea was that InvalidBlocks could contain invalid blocks. but it doesnt mean they have to. 
although it is unexpected.
